doc/admin-guide/logging/formatting.en.rst

```diff
@@ -320,6 +325,33 @@ ssh            essh
 cssh           ecssh
 ============== ===================
 
+It is also possible to log all of the headers in a transaction message with a
+single field.  For each original original field, there is a variant which ends in
+``ah`` rather than ``h``, as shown here:
+
+============== ===================
+Original Field All Headers Variant
+============== ===================
+cqh            cqah
+pqh            pqah
+psh            psah
+ssh            ssah
+cssh           cssah
+============== ===================
+
+No particular header is specified when using these variants, for example::
+
+    Format = '%<cqah>'
+
+The output generated by these fields has the pattern::
+
+    {{{tag1}:{value1}}{{tag2}:{value2}}...}
+
+(The size of some messages may exceed internal buffer capacity.  This may
+result in the value of the last header being truncated, in which case, the
+value will end with ``...}``.  This may also result in the ommission of
+entire tag/value pairs.)
```

Comment:
Note that if the log entry as a whole exceeds proxy.config.log.log_buffer_size, you get nothing in the log, at all.